test/components/entity/diff/table.spec.js

```diff
@@ -0,0 +1,111 @@
+  });
+  const entityVersion = last(container.requestData.localResources.entityVersions);
+  return mount(EntityDiffTable, {
+    props: { diff: entityVersion[options?.props?.diff ?? 'baseDiff'] },
+    global: {
+      provide: { projectId: '1', datasetName: 'trees', uuid, entityVersion }
+    },
+    container
+  });
+};
+
+describe('EntityDiffTable', () => {
+  describe('Comparing row', () => {
+    const createConflict = () => {
+      const users = Array.from(testData.extendedUsers
+        .createPast(1, { displayName: 'Alice' })
+        .createPast(1, { displayName: 'Bob' })
+        .createPast(1, { displayName: 'David' }));
+      testData.extendedEntities.createPast(1, { creator: users[0] });
+      testData.extendedEntityVersions
+        .createPast(1, { baseVersion: 1, creator: users[1] })
+        .createPast(1, { baseVersion: 1, creator: users[2] });
+    };
+
+    it('does not show the row if the version is not a conflict', () => {
+      testData.extendedEntities.createPast(1);
+      testData.extendedEntityVersions.createPast(1);
+      mountComponent().find('.conflicting').exists().should.be.false();
+    });
+
+    it('shows the old version number', () => {
+      createConflict();
+      const table = mountComponent();
+      const td = table.get('td:nth-child(2)');
+      td.text().should.equal('v1 (Update by Alice)');
+      td.getComponent(EntityVersionLink).props().version.version.should.equal(1);
+    });
+
+    it('shows the correct version number for the server diff', () => {
+      createConflict();
+      const table = mountComponent({
+        props: { diff: 'serverDiff' }
+      });
+      table.get('td:nth-child(2)').text().should.equal('v2 (Update by Bob)');
+    });
+
+    it('shows the new version number', () => {
+      createConflict();
+      const table = mountComponent();
+      const td = table.get('td:last-child');
+      td.text().should.equal('v3 (Update by David)');
+      td.getComponent(EntityVersionLink).props().version.version.should.equal(3);
+    });
+  });
+
+  it('renders an EntityDiffRow for each change', () => {
+    testData.extendedEntities.createPast(1, {
+      label: 'dogwood',
+      data: { height: '1', circumference: '2' }
+    });
+    testData.extendedEntityVersions.createPast(1, {
+      label: 'Dogwood',
+      data: { height: '3', circumference: '4' }
+    });
+    const table = mountComponent();
+    table.findAllComponents(EntityDiffRow).length.should.equal(3);
+  });
+
+  it('passes correct props to EntityDiffRow for a label change', () => {
+    testData.extendedEntities.createPast(1, { label: 'dogwood' });
+    testData.extendedEntityVersions.createPast(1, { label: 'Dogwood' });
+    const table = mountComponent();
+    const props = table.getComponent(EntityDiffRow).props();
+    props.oldVersion.version.should.equal(1);
+    props.name.should.equal('label');
+  });
+
+  it('passes correct props to EntityDiffRow for a property change', () => {
```

Comment:
Remove this test? There's not a difference between labels and properties in this component.